pysand/erosion.py

```diff
@@ -1,7 +1,32 @@
 import numpy as np
+import pandas as pd
+import logging
+import pysand.exceptions as exc
 
+logger = logging.getLogger(__name__)
 # Models from DNVGL RP-O501, equation references in parenthesis
 
+
+def validate_inputs(v_m, Q_s, rho_m=30, mu_m=1.5e-5, GF=3, D=0.5, d_p=1):
```

Comment:
You should probably do: 
`def validate_inputs(arguments):`
then later in function:
`if 'd_p' in arguments:` and then validate d_p.
This way the validation function can be general and you can have all validation here!